config/config.go

```diff
@@ -30,6 +30,7 @@ type Config struct {
 	Verbosity               int
 	RawMetadata             string
 	AgentTTL                string
+	FollowOnly              bool
```

Comment:
[ Quoting <notifications@github.com> in "Re: [fleet] Implement follow_only f..." ]
>> @@ -30,6 +30,7 @@ type Config struct {
>>  	Verbosity               int
>>  	RawMetadata             string
>>  	AgentTTL                string
>> +	FollowOnly              bool
>
>How about we simplify this down to `NoScheduler` or something?

NoLeader? But I dislike the "No" part, so "NeverLeader" ?

/Miek

--
Miek Gieben